flow/project.py

```diff
@@ -1843,24 +1974,40 @@ def _print_progress(x):
         except KeyError:
             cached_status = dict()
 
-        _get_job_status = functools.partial(self.get_job_status,
-                                            ignore_errors=ignore_errors,
-                                            cached_status=cached_status)
+        _get_job_labels = functools.partial(self._get_job_labels,
+                                            ignore_errors=ignore_errors)
+
+        _get_group_status = functools.partial(self._get_group_status,
+                                              jobs=jobs,
+                                              ignore_errors=ignore_errors,
+                                              cached_status=cached_status)
+
+        groups = []
+        for group in self._groups.values():
+            if len(group.operations) == 1:
+                for name in group.operations:
```

Comment:
if the length is 1 why do you need to iterate over it? or is it nested?

Reply:
This is nested to I had to iterate over it.